lib/changeable.rb

```diff
@@ -14,6 +14,10 @@ def changeable?(user)
     elsif user.has_role?("admin", "manager")
       true
 
+    # project admins can change the projects they are admins of
+    elsif (self.class == Admin::Project) && user.is_project_admin? && user.admin_for_projects.include?(self)
+      true
+
```

Comment:
Ok.  I added this to the Admin::Project model and removed the else from changeable:

    def changeable?(user)
      # project admins can change the projects they are admins of
      if user.is_project_admin? && user.admin_for_projects.include?(self)
        true
      else
        super(user)
      end
    end